File: 02_algorithm/programmers/Level2/JadenCase 문자열 만들기.py
def solution(data):
    data = data.lower()
    words = data.split(' ')
    answer = ''
    for word in words:
        word = word.capitalize()
        answer += word + ' '
    return answer[:-1]

print(solution('3people unFollowed me'))
print(solution('for the last week'))

File: 06_js/04_algorithm/programmers/Level2/JadenCase 문자열 만들기.js
function solution(s) {
    let answer = '';
    for (let i = 0; i < s.length; i++) {
        if (i === 0 || s[i-1] === ' ') {
            answer += s[i].toUpperCase();
        } else {
            answer += s[i].toLowerCase();
        }
    }
    return answer;
}